src/components/navigation/navigation.stories.svelte

```diff
@@ -0,0 +1,102 @@
+<script lang="ts">
+  import { Meta, Story, Template } from '@storybook/addon-svelte-csf'
+
+  import Navigation from './navigation.svelte'
+  import NavigationItem from './navigationItem.svelte'
+  import NavigationHeader from './navigationHeader.svelte'
+  import NavigationActions from './navigationActions.svelte'
+  import NavigationSeparator from './navigationSeparator.svelte'
+  import NavigationMenu from './navigationMenu.svelte'
+  import Icon from '../icon/icon.svelte'
+  import SlotInfo from '../../storyHelpers/SlotInfo.svelte'
+  import Slot from '../../storyHelpers/Slot.svelte'
+  import Toggle from '../toggle/toggle.svelte'
+</script>
+
+<Meta
+  title="Components/Navigation"
+  component={Navigation}
+  argTypes={{
+    kind: {
+      table: {
+        disable: true
+      }
+    }
+  }}
+  args={{}}
+/>
+
+<Template let:args>
+  <Navigation {...args}>
+    <NavigationHeader slot="header">
+      <Icon slot="logo" name="social-brave-release-favicon-fullheight-color" />
+      <h1><span class="logo-mark">Brave</span> Accounts</h1>
+    </NavigationHeader>
+
+    <NavigationMenu>
+      {#each [1, 2, 3, 4, 5] as item}
+        <NavigationItem
+          href="#"
+          icon="browser-ntp-widget"
+          isCurrent={item === 3}>Item {item}</NavigationItem
+        >
+      {/each}
+      <NavigationSeparator />
+      <NavigationItem href="#" icon="browser-ntp-widget">
+        Item with sub nav
+        <NavigationMenu slot="subnav">
+          <NavigationItem href="#" icon="agenda" isCurrent={true}>
+            Testing
+          </NavigationItem>
+          <NavigationItem href="#" icon="browser-ntp-widget">
+            Item with sub nav
+            <NavigationMenu slot="subnav">
+              <NavigationItem href="#" icon="agenda" isCurrent={true}>
+                Testing
+              </NavigationItem>
+            </NavigationMenu>
+          </NavigationItem>
+        </NavigationMenu>
+      </NavigationItem>
+    </NavigationMenu>
+
+    <NavigationActions slot="actions">
+      <Toggle>
```

Comment:
Okay, in the story this looks kinda odd because it's just sitting there by itself - should it look more connected to the rest of the menu?